Makefile

```diff
@@ -1,2 +1,56 @@
+all: whatdoesitdo tests authors
+
 whatdoesitdo:
+	@# generate the README "What does it do?" section
 	@./gen-readme.sh
+
+# To decrease tests verbosity, comment out unneeded targets
+tests: downloadffprefs checknotcovered checkdeprecated stats cleanup
```

Comment:
Yeah I saw that post also when I was looking at the subject. Is it so, that everything below that will not run as parallel or what? I mean there are still some things, that can be run as parallel in the makefile, e.g. ```authors```, ```whatdoesitdo``` and all the rest for that matter, once ```downloadffprefs``` has finished.